myuw/static/js/card/schedule/visual.js

```diff
@@ -6,9 +6,9 @@ var VisualScheduleCard = {
     should_display_card: function() {
         if (!window.user.student ||
             !window.card_display_dates.is_before_last_day_of_classes) {
```

Comment:
Is this check really needed?
if (!window.force_visual_schedule_display) {
                return false;
            }